### ISSUE TITLE
Boxes at the images are being removed as soon as they appear!

### ISSUE THREAD

**Issue by sudipt1999** [sudipt1999/FaceRecApp#2]
Boxes on the image are disappeared as soon as they are shown! kindly make them fix until the user dosen't change url.

**Comment by ranndev** [comment]
Looking on this now.

**Comment by sudipt1999** [comment]
Sure !Thanks for contributing!

**Comment by sudipt1999** [comment]
@rannie-peralta  are you working on this!

**Comment by ranndev** [comment]
Yes @sudipt1999 , I'm still interested to work on this. I'm just waiting for the #3 to be done first.

**Comment by sudipt1999** [comment]
Sorry my bad

**Comment by AnshulGoyal589** [comment]
@sudipt1999 if the issue is still open, kindly assign the issue to me
